src/chart/generateCategoricalChart.tsx

```diff
@@ -239,11 +239,31 @@ const getTooltipData = (state: CategoricalChartState, chartData: any[], layout:
     const activePayload = getTooltipContent(state, chartData, activeIndex, activeLabel);
     const activeCoordinate = getActiveCoordinate(layout, ticks, activeIndex, rangeData);
 
+    const { x: mouseX, y: mouseY } = rangeData;
+    const getActiveEntry = (o: object) => _.get(o, ['props', 'data', activeIndex]);
+    let activeChild = _.find(formatedGraphicalItems, entry => {
+      const activeEntry = getActiveEntry(entry);
+      if (activeEntry) {
+        const { x, width, y, height } = activeEntry;
```

Comment:
this will only works for rectangle coordinate charts, such as line, bar.
How about pie-chart? radial-bar-chart?

Reply:
We currently don't need this data for pie-chart and radial-bar-chart, because there's not a child/sub-element that we need to know about for the tooltip, so that's why it's not an issue for our use case, but if there's a better way to handle this, then I'd definitely be open to using it instead.

Reply:
Can you show more detail about your case?

Reply:
The current tooltip on a stacked bar chart shows all of the values and we'd like to show just the value being hovered over. Currently, there's not a way to tell which one is being hovered over and the intent of this change is to provide that information. If there's a better way to accomplish that, then we're definitely open to that, but this was one approach to doing that.

Reply:
OK! Got it! I will think about this feature.